service/pom.xml

```diff
@@ -15,9 +15,9 @@
 	<properties>
 		<jts.version>1.13</jts.version>
 
-		<docker.dev.start.phase>test-compile</docker.dev.start.phase>
-		<docker.liquibase.phase>test-compile</docker.liquibase.phase>
-		<docker.dev.prestop.phase>validate</docker.dev.prestop.phase>
+<!--		<docker.dev.start.phase>test-compile</docker.dev.start.phase>-->
```

Comment:
Remove alltogether?

Reply:
Nope, because we will add arquillian tests soon(tm) and then these are needed